sdl_android/src/androidTest/java/com/smartdevicelink/transport/SdlRouterServiceTests.java

```diff
@@ -0,0 +1,85 @@
+package com.smartdevicelink.transport;
+
+
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.os.Message;
+import android.test.AndroidTestCase;
+import android.util.Log;
+
+import com.smartdevicelink.protocol.SdlPacket;
+
+import junit.framework.Assert;
+
+import java.lang.ref.WeakReference;
+
+public class SdlRouterServiceTests extends AndroidTestCase {
+
+    public static final String TAG = "SdlRouterServiceTests";
+
+    @Override
+    protected void setUp() throws Exception {
+        super.setUp();
+    }
+
+    @Override
+    protected void tearDown() throws Exception {
+        super.tearDown();
+        //Nothing here for now
+    }
+
+    /**
+     * Test null bundle handling in AltTransportHandler when handling messages. Only test the case of
+     * msg.what == TransportConstants.ROUTER_RECEIVED_PACKET
+     */
+    public void testAlTransportHandlerHandleNullBundle() {
+        Looper.prepare();
```

Comment:
you need to check if a looper is already available, otherwise it will error:
```java
        if (Looper.myLooper() == null) {
            Looper.prepare();
        }
```